data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,403 @@ mission "Unfettered returning home"
+	to complete
+		not "unfettered: fled with the relic"
+		# Needed to make sure the mission doesn't auto complete if no ships spawn.
+		has "flagship model: Anomalocaris"
+	on complete
+		clear "old reputation: Hai (Unfettered)"
+		log "Managed to use an ancient Hai warship to beat an Unfettered challenge, earning respect with the Unfettered Hai so long as the ship remains in your possession."
+		conversation
+			`When you land in one piece, Mera approaches, looking quite surprised. "You really won! I saw everything, and the data I got on the effects of these exotic weapons will be quite useful for my research. Was it hard for you?"`
+			choice
+				`	"It was fun."`
+					goto next
+				`	"This was easy!"`
+				`	"It was a challenge."`
+					goto challenge
+			`	"Don't get overconfident, Captain," she muses. "I am sure you know what happens to such pilots.`
+				goto next
+			label challenge
+			`	"Oh, so their plan almost worked after all? Maybe I should have participated in the fight too."`
+			label next
+			`	"Well, congratulations, I did not think you had it in you. I will give you authorization to land, on the condition that you keep this relic from our past in one piece. Remember, right now, your reputation with us is solely due to to that ship. If you somehow lose it because of incompetence on your own end, it would look pretty bad to us true Hai!"`
+			`	Mera then walks off as she begins reviewing data from your fight on a small device. For now, it seems that the Unfettered Hai have some sort of respect for you, so long as you have the Anomalocaris.`
+
+
+
+mission "Undeserved Anomalocaris"
+	minor
+	invisible
+	landing
+	source
+		attributes "unfettered"
+	to offer
+		has "ship model: Anomalocaris"
+		"reputation: Hai (Unfettered)" < 0
+		not "Hai Legacy 2: active"
+	on offer
+		conversation
+			`You land on this Unfettered world in possession of the Anomalocaris, and, as you do, countless Hai hastily head to your ship, each armed with several weapons. One of them points at you and makes a declaration in front of the crowd that is quickly forming, shouting loud enough for all to hear. "You do not deserve to fly this relic of old!"`
+			`	One standing behind them then chimes in. "You wasted your opportunity for a glorious battle to earn that right! We know of what you've failed!"`
+				to display
+					has "Hai Legacy 3: offered"
+			`	A more important-looking Unfettered then steps up close to you, speaking in a menacing tone. "You must surrender the ship to us immediately, along with the weapons of the savages, if you did not already rob us of those." For a moment, they look down in thought, then speak to you once more. "We will give you another ship so you can leave, but you must leave the Anomalocaris with us!"`
+
+			choice
+				`	(Hand over the Anomalocaris.)`
+				`	(Decline their request and leave.)`
+					depart
```

Comment:
```suggestion
					flee
```

Reply:
if I do flee it'll decline the mission which wont offer again